functions/v2/helloworld/hello-gcs/src/main/java/functions/eventpojos/GcsEvent.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package functions.eventpojos;
+
+import java.util.Date;
+
+public class GcsEvent {
```

Comment:
I assume that the CloudEvent type libraries won't work here?

(IIUC, they were supposed to obviate the need for these POJOs.)

Reply:
We aren't using the libraries. They are not GA